composer.json

```diff
@@ -9,10 +9,20 @@
     },
     "require-dev": {
         "brain/monkey": "^2",
+        "brainmaestro/composer-git-hooks": "^2.8",
         "dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
         "mockery/mockery": "^1.2.4",
         "phpcompatibility/phpcompatibility-wp": "2.1.0",
         "squizlabs/php_codesniffer": "^3.4",
         "wp-coding-standards/wpcs": "2.2.0"
+    },
+    "extra": {
+        "hooks": {
+            "pre-commit": "bash bin/pre-commit.sh"
+        }
+    },
+    "scripts": {
+        "post-install-cmd": "cghooks add --ignore-lock",
+        "post-update-cmd": "cghooks update"
```

Comment:
I think this means that on running `composer install`, `cghooks` will install the pre-commit hook.